base/shell/cmd/dir.c

```diff
@@ -1782,6 +1782,10 @@ ResolvePattern(
             pszPatternPart = NULL;
         }
     }
+    else if (_istalpha(pNextDir[0]) && pNextDir[1] == TEXT(':') && pNextDir[2] == TEXT('\0'))
+    {
+        /* Drive name only. Nothing to do. */
```

Comment:
So to clarify this comment, the syntax "<drive_letter>:" without any trailing slash actually means: "current directory on this drive".

Reply:
OK.